modules/protocol-tests/resources/META-INF/smithy/test-config.json

```diff
@@ -1,1592 +1,477 @@
+          },
+          {
+            "id": "DocumentTypeAsPayloadOutput"
+          },
+          {
+            "id": "DocumentTypeAsPayloadOutputString"
+          },
+          {
+            "id": "RestJsonSerializeNumberUnionValue"
+          },
+          {
+            "id": "RestJsonSerializeListUnionValue"
+          },
+          {
+            "id": "RestJsonSerializeBooleanUnionValue"
+          },
+          {
+            "id": "RestJsonSerializeStringUnionValue"
+          },
+          {
+            "id": "RestJsonSerializeStructureUnionValue"
+          },
+          {
+            "id": "RestJsonSerializeEnumUnionValue"
+          },
+          {
+            "id": "RestJsonSerializeMapUnionValue"
+          },
+          {
+            "id": "RestJsonDeserializeListUnionValue"
+          },
+          {
+            "id": "RestJsonDeserializeMapUnionValue"
+          },
+          {
+            "id": "RestJsonDeserializeNumberUnionValue"
+          },
+          {
+            "id": "RestJsonDeserializeStructureUnionValue"
+          },
+          {
+            "id": "RestJsonDeserializeBooleanUnionValue"
+          },
+          {
+            "id": "RestJsonDeserializeStringUnionValue"
+          },
+          {
+            "id": "RestJsonDeserializeEnumUnionValue"
+          },
+          {
+            "id": "PostUnionWithJsonNameRequest1"
+          },
+          {
+            "id": "PostUnionWithJsonNameRequest3"
+          },
+          {
+            "id": "RestJsonInputUnionWithUnitMember"
+          },
+          {
+            "id": "PostUnionWithJsonNameRequest2"
+          },
+          {
+            "id": "RestJsonOutputUnionWithUnitMember"
+          },
+          {
+            "id": "PostUnionWithJsonNameResponse1"
+          },
+          {
+            "id": "PostUnionWithJsonNameResponse2"
+          },
+          {
+            "id": "PostUnionWithJsonNameResponse3"
+          },
+          {
+            "id": "RestJsonEndpointTrait"
+          },
+          {
+            "id": "RestJsonTestBodyStructure"
+          },
+          {
+            "id": "RestJsonTestPayloadStructure"
+          },
+          {
+            "id": "RestJsonEndpointTraitWithHostLabel"
+          },
+          {
+            "id": "RestJsonHttpWithEmptyBody"
+          },
+          {
+            "id": "RestJsonNoInputAndOutputAllowsAccept"
+          },
+          {
+            "id": "RestJsonEmptyInputAndEmptyOutputWithJson"
+          },
+          {
+            "id": "RestJsonUnitInputAllowsAccept"
+          },
+          {
+            "id": "RestJsonNoInputAllowsAccept"
+          },
+          {
+            "id": "RestJsonHttpRequestWithLabelsAndTimestampFormat"
+          },
+          {
+            "id": "RestJsonHttpRequestLabelEscaping"
+          },
+          {
+            "id": "RestJsonInputAndOutputWithStringHeaders",
+            "appliesTo": "client",
+            "testType": "response"
```

Comment:
I'm assuming if those two aren't specified they fallback to `appliesTo`: both server and client and `testType`  both request and response

Reply:
your assumption is correct